README.md

```diff
@@ -8,362 +8,63 @@ Blobfuse2 is stable and ***supported by Microsoft*** when used within its [docum
 
 Please submit an issue [here](https://github.com/azure/azure-storage-fuse/issues) for any issues/feature requests/questions.
 
-[This](#config-guide) section will help you choose the correct config for Blobfuse2.
+## [List of Supported Platforms](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Supported-Platforms)
+
+## [Steps to Install Blobfuse2](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Installation)
+
+## [Choose config for Blobfuse2](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Config-Guide)
+
+## [Blockcache Limitations And Recommendations](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Blockcache-Limitations-And-Recommendations)
+
+## [Commands to use BlobFuse2](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Usage)
+
+## [Blobfuse2 Benchmarks](https://azure.github.io/azure-storage-fuse/)
+
+## [Features of BlobFuse2](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Features)
+
+## [_New BlobFuse2 Health Monitor_](https://github.com/Azure/azure-storage-fuse/blob/main/tools/health-monitor/README.md)
+
+## [Supported Operations](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Cli%E2%80%90Parameters)
+
+## [CLI parameters](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Cli%E2%80%90Parameters)
+
+## [Environment variables](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Environment-Variables)
+
+## [Blob Filter](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Blob-Filter)
+
+## [Preload Data in Blobfuse2](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2%E2%80%90Preload)

```

Comment:
As this is blobfuse2 readme, shall we remove 'blobfuse2' from most of these links for e.g, instead of "Blobfuse2 health monitor" we can just say "Health monitor"